lib/src/data/datasource/remote/api_service.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+
+import '../../models/genre_response.dart';
+import '../../models/movie_response.dart';
+import 'i_api_service.dart';
+
+class ApiService implements IApiService {
+  static const String _apiKey = '025aa98c350d497beb053d9b5e169637';
+  static const String _baseUrl = 'https://api.themoviedb.org';
```

Comment:
You can add also to the baseUrl the `/3/`

Reply:
done